### DISCUSSION TITLE
Deal line of project

### DISCUSSION THREAD

**Discussion by YuriyMorozyuk95** [microsoft/RAG_Hack#67] [Q&A] [answered]
In order to work on the project, we need to spend time during the nights after work and on weekends. It is possible to upload the video and repository by 1:00 PM (13:00) PT on September 15. This corresponds to 11:00 PM (23:00) on September 15 in Tallinn time. Can we at least upload the video by that time?

**Comment by pamelafox** [comment, marked as answer]
Hi @YuriyMorozyuk95 , Those both sound like they're before the deadline, which is PT:
https://github.com/microsoft/RAG_Hack?tab=readme-ov-file#project-submission

**Reply by YuriyMorozyuk95** [reply]
Thanks, you a lot, Pamela, and sorry for miss understanding. Firstly, I found that streams end 13 of September, and thought that it is deadline(
